lib/src/participant/local.dart

```diff
@@ -148,6 +148,21 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
     publishOptions ??=
         track.lastPublishOptions ?? room.roomOptions.defaultVideoPublishOptions;
 
+    // if no encoding is specified, use the default encoding
+    if (publishOptions.videoEncoding == null) {
```

Comment:
is it possible for it to be null here? It seems that room.roomOptions.defaultVideoPublishOptions should always be initialized to something